src/generic/scrlwing.cpp

```diff
@@ -984,6 +984,14 @@ void wxScrollHelperBase::HandleOnMouseLeave(wxMouseEvent& event)
         if ( !m_targetWindow->HasScrollbar(orient) )
             return;
 
+#ifdef __WXMSW__
```

Comment:
Is this really MSW-specific? It looks like we might need to do it for all ports...

Reply:
I think the code in `wxAutoScrollTimer::Notify()` doesn't handle RTL windows correctly. But this still doesn't explain why it works in wxGTK.

IMO it would be highly desirable to understand what is really going on here and where does the discrepancy in behaviour between different ports really come from in order to fix it at the source.

Reply:
Indeed; the problem was in `wxWindowMSW::GenerateMouseLeave()` fixed now by 
b419237